_includes/includes/ui/keyboard-details.php

```diff
@@ -168,11 +177,14 @@ protected static function WriteWebBoxes($useDescription) {
       $url = KeymanHosts::Instance()->keymanweb_com ."/#$lang,Keyboard_" . self::GetWebKeyboardId();
       if($useDescription) {
         $description = htmlentities(self::$keyboard->name);
-        $description = "<div class=\"download-description\">Use $description in your web browser. No need to install anything.</div>";
-        $linktext = 'Use keyboard online';
+        $useKeyboardDescription = _m("use_keyboard_button_description",
+          $description);
+        $description = "<div class=\"download-description\">
+          $useKeyboardDescription</div>";
```

Comment:
```suggestion
        $description = "<div class='download-description'>"._m("use_keyboard_button_description", $description)."</div>";
```

Reduce the number of variables